pkg/controller/health_check.go

```diff
@@ -35,8 +35,8 @@ func (srv *HealthCheckService) getHealthInfo() map[string]interface{} {
 	return m
 }
 
-// HealthCheckHandler returns a default heath check result.
-func (srv *HealthCheckService) GetHealthCheckHandler(ctx *gin.Context) {
+// GetHealthCheckHandler returns a default heath check result.
+func (srv *HealthCheck) GetHealthCheckHandler(ctx *gin.Context) {
```

Comment:
you could probably rename this func to just `Handler`, so when you register it in the server (see https://github.com/codeready-toolchain/registration-service/pull/25/files#diff-0fc56aea4678c02bd6ecd92471b1e16cR34), you'll have something like this:
```
handler := gin.HandlerFunc(healthCheck.Handler)
```

Reply:
hmm i'm not sure that would work - if we have more than one handler func this name would need to be changed later

Reply:
`healthCheck.Get` would work I think but what about `GetHandler()` instead?
So we would have something like:
```
healthCheckCtrl := controller.NewHealthCheck(srv.logger, srv.Config())
signupCtrl := controller.NewSignup(srv.logger, srv.Config())

v1.GET("/health", healthCheckCtrl.GetHandler)
v1.GET("/signup", signupCtrl.GetHandler)
v1.POST("/signup", signupCtrl.PostHandler)
```
But just 'Get()', `Post()` would work too I think...

Reply:
fixed here: b45b29c341cf3a376237e278fc281b4a5d0dfdc2